localstack/services/route53resolver/models.py

```diff
@@ -15,94 +20,151 @@ def __init__(self):
         self.firewall_domains = {}
         self.firewall_rules = {}
         self.firewall_rule_group_associations = {}
-
-
-## helper functions for the backend
-def get_firewall_rule_group(id):
-    """returns firewall rule group with the given id if it exists"""
-    region_details = Route53ResolverBackend.get()
-    firewall_rule_group = region_details.firewall_rule_groups.get(id)
-    if not firewall_rule_group:
-        raise ResourceNotFoundException(
-            f"Can't find the resource with ID '{id}'. Trace Id: '{aws_stack.get_trace_id()}'"
+        self.resolver_query_log_configs = {}
```

Comment:
nit: One more small comment - would be great if we could add type hints (and a short description) for these member variables, similar to what we have for `firewall_rule_groups` in line 15 above. 

We could also pick that up in one of the next PRs @macnev2013 .. 👍